logic/src/commonTest/kotlin/com/wire/kalium/logic/sync/slow/SlowSyncManagerTest.kt

```diff
@@ -412,6 +465,20 @@ class SlowSyncManagerTest {
                 .thenReturn(duration)
         }
 
+        fun withOldLastSlowSyncVersion() = apply {
+            given(slowSyncRepository)
+                .suspendFunction(slowSyncRepository::getSlowSyncVersion)
+                .whenInvoked()
+                .thenReturn(0)
+        }
+
+        fun withNewLastSlowSyncVersion() = apply {
+            given(slowSyncRepository)
+                .suspendFunction(slowSyncRepository::getSlowSyncVersion)
+                .whenInvoked()
+                .thenReturn(Int.MAX_VALUE)
+        }
```

Comment:
Just the naming of these functions are ambiguous. They are talking about the last slow sync performed, right? Not the CURRENT Slow Sync version.

```suggestion
        fun withLastSlowSyncPerformedOnAnOldVersion() = apply {
            given(slowSyncRepository)
                .suspendFunction(slowSyncRepository::getSlowSyncVersion)
                .whenInvoked()
                .thenReturn(0)
        }

        fun withLastSlowSyncPerformedOnANewVersion() = apply {
            given(slowSyncRepository)
                .suspendFunction(slowSyncRepository::getSlowSyncVersion)
                .whenInvoked()
                .thenReturn(Int.MAX_VALUE)
        }
```